{"level":"ERROR","time":"2020-05-16T11:45:50.858+0530","caller":"MessageService/main.go:80","message":"context deadline exceeded"}
{"level":"ERROR","time":"2020-05-16T11:45:57.541+0530","caller":"MessageService/main.go:191","message":"dial tcp [::1]:19092: connectex: No connection could be made because the target machine actively refused it."}
{"level":"ERROR","time":"2020-05-16T12:16:08.267+0530","caller":"MessageService/main.go:80","message":"context deadline exceeded"}
{"level":"ERROR","time":"2020-05-16T12:16:14.943+0530","caller":"MessageService/main.go:192","message":"dial tcp [::1]:19092: connectex: No connection could be made because the target machine actively refused it."}
{"level":"INFO","time":"2020-05-20T20:05:48.055+0530","caller":"MessageService/main.go:258","message":"metadata","Topic":"email","Key":"14244","Offset":150}
{"level":"INFO","time":"2020-05-20T20:05:48.059+0530","caller":"MessageService/main.go:259","message":"{\"Email\":\"sunilsn@iitk.ac.in\",\"Phone\":\"9916194654\",\"MessageBody\":\"hellohiytttty\",\"Transactionid\":\"125456456\",\"Customerid\":\"347656r76789\",\"Key\":\"14244\",\"partition\":-1}"}
{"level":"ERROR","time":"2020-05-20T20:06:09.075+0530","caller":"MessageService/main.go:199","message":"dial tcp 74.125.24.109:587: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"level":"ERROR","time":"2020-05-20T20:06:11.076+0530","caller":"MessageService/main.go:97","message":"connection(localhost:27017[-9]) incomplete read of message header: read tcp 127.0.0.1:63154->127.0.0.1:27017: i/o timeout"}
{"level":"INFO","time":"2020-05-20T20:06:11.080+0530","caller":"MessageService/main.go:258","message":"metadata","Topic":"email","Key":"","Offset":151}
{"level":"INFO","time":"2020-05-20T20:06:11.080+0530","caller":"MessageService/main.go:259","message":"{\"Email\":\"sunilsn@iitk.ac.in\",\"Phone\":\"9916194654\",\"MessageBody\":\"hellohiytttty\",\"Transactionid\":\"125456456\",\"Customerid\":\"347656r76789\",\"Key\":\"\",\"partition\":-1}"}
{"level":"ERROR","time":"2020-05-20T20:06:32.086+0530","caller":"MessageService/main.go:199","message":"dial tcp 74.125.24.109:587: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"level":"INFO","time":"2020-05-20T20:06:32.696+0530","caller":"MessageService/main.go:99","message":"Succesfully Inserted"}
{"level":"INFO","time":"2020-05-20T20:06:32.700+0530","caller":"MessageService/main.go:258","message":"metadata","Topic":"email","Key":"","Offset":152}
{"level":"INFO","time":"2020-05-20T20:06:32.700+0530","caller":"MessageService/main.go:259","message":"{\"Email\":\"sunilsn@iitk.ac.in\",\"Phone\":\"9916194654\",\"MessageBody\":\"hellohiytttty\",\"Transactionid\":\"125456456\",\"Customerid\":\"347656r76789\",\"Key\":\"\",\"partition\":0}"}
{"level":"ERROR","time":"2020-05-20T20:06:53.706+0530","caller":"MessageService/main.go:199","message":"dial tcp 74.125.24.109:587: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"level":"INFO","time":"2020-05-20T20:06:54.314+0530","caller":"MessageService/main.go:99","message":"Succesfully Inserted"}
